examples/deadline/Local-Zone/ts/bin/config.ts

```diff
@@ -38,14 +38,14 @@ class AppConfig {
    * "10.1.16"
    * @default 10.1.20 is used, to match the worker AMI ID provided below
```

Comment:
Default needs updating
```suggestion
   * @default 10.3.1 is used, to match the worker AMI ID provided below
```